defaults/main.yml

```diff
@@ -8,20 +9,29 @@ xen_vman_iscsi_logout_timeout:          40
 xen_vman_mount_timeout:                 20
 xen_vman_umount_timeout:                20
 
+xen_vman_iscsi_server:                  ""
```

Comment:
But otherwise it would be silly if I only use NFS

Reply:
I don't think the value is mandatory when only NFS is used. Therefore please omit the default. This way, ansible will fail straight-forward when it is not defined, instead of templating services that will fail only on runtime because of their wrong addresses.